lib/WebService/PayPal/PaymentsAdvanced/Response/FromSilentPOST.pm

```diff
@@ -45,29 +60,22 @@ has _ip_address_is_verified => (
     builder  => '_build_ip_address_is_verified',
 );
 
-# Payments Advanced IPs listed at
-# https://www.paypal-techsupport.com/app/answers/detail/a_id/883/kw/payflow%20Ip
-
-has _ip_addresses => (
-    is          => 'ro',
-    isa         => ArrayRef,
-    handles_via => 'Array',
-    handles     => { _all_verified_ip_addresses => 'elements' },
-    default     => sub {
-        [ '173.0.81.1', '173.0.81.33', '66.211.170.66', '173.0.81.65' ];
-    },
-);
-
 with(
     'WebService::PayPal::PaymentsAdvanced::Role::HasTender',
     'WebService::PayPal::PaymentsAdvanced::Role::HasTokens',
     'WebService::PayPal::PaymentsAdvanced::Role::HasTransactionTime',
 );
 
+# Payments Advanced IPs listed at
+# https://www.paypal-techsupport.com/app/answers/detail/a_id/883/kw/payflow%20Ip
```

Comment:
This should be https://www.paypal.com/us/selfhelp/article/what-are-the-ip-addresses-for-payflow-servers-ts1465/1

Reply:
Thanks!